bin/ch-test

```diff
@@ -111,12 +111,20 @@ builder_set () {
 dir_img_mk () {
     dir_img_rm
     printf "creating %s\n\n" "$CH_TEST_IMGDIR"
-    mkdir "$CH_TEST_IMGDIR"
+    if [[ -n $SLURM_JOBID ]]; then
+        srun --ntasks-per-node=1 mkdir "$CH_TEST_IMGDIR"
+    else
+        mkdir "$CH_TEST_IMGDIR"
+    fi
```

Comment:
The unpack assumptions are not squashfs friendly.

Reply:
Are they? Creating and removing `$CH_TEST_IMGDIR` will eventually (not now) be redundant for SquashFS, but it won't be harmful.